src/helpers/web.ts

```diff
@@ -132,7 +133,12 @@ function camelToSnake(str: string): string {
  * @returns {string}
  */
 export function generateQuery(queryParams: IServiceParams): string {
+  checkValueType('pr', queryParams.pr, 'number')
```

Comment:
out of scope likely, but I'd rather us do a `validateQuery` function that this function calls

Reply:
100%. I didn't want to get distracted, this was messy enough :)